res/mac/FreenetTray/src/defaults.plist

```diff
@@ -2,8 +2,12 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>firstlaunch</key>
+	<true/>
 	<key>nodepath</key>
 	<string>$INSTALL_PATH</string>
+	<key>nodefcpurl</key>
+	<string>tcp://127.0.0.1:9481</string>
```

Comment:
Is there a mechanism to update this if the installer changes the port from the default? I couldn't find a place where the FProxy port was updated either. Am I missing something?

Is it safe to assume there will always be an IPv4 loopback interface? Would `localhost` be safer?